iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractConnectable.java

```diff
@@ -25,16 +25,22 @@
 
     protected final List<TerminalExt> terminals = new ArrayList<>();
     private final Ref<NetworkImpl> networkRef;
-    private boolean removed = false;
+    protected boolean removed = false;
 
     AbstractConnectable(Ref<NetworkImpl> ref, String id, String name, boolean fictitious) {
         super(id, name, fictitious);
         this.networkRef = Objects.requireNonNull(ref);
     }
 
-    void addTerminal(TerminalExt terminal) {
+    void addTerminal(TerminalExt terminal, boolean setConnectable) {
```

Comment:
what is the purpose of this new parameter? Seems to be not used.

Reply:
Similarly, this is a residue from an old commit, when TieLine was still extending Branch, and was sharing a terminal with the DanglingLine. Not useful anymore, removing it.